translate/storage/test_jsonl10n.py

```diff
@@ -70,6 +70,14 @@ def test_serialize(self):
 
         assert out.getvalue() == b'{\n    "key": {\n        "message": "value",\n        "description": "note"\n    }\n}\n'
 
+    def test_serialize_no_description(self):
+        store = self.StoreClass()
+        store.parse('{"key": {"message": "value"}}')
+        out = BytesIO()
+        src = store.serialize(out)
+
+        assert out.getvalue() == b'{\n    "key": {\n        "message": "value"\n    }\n}\n'
```

Comment:
Nit: I would avoid having `\n` and instead use string in triple quotes to make it more readable.